module/support/ReactionStats/src/ReactionStats.cpp

```diff
@@ -0,0 +1,46 @@
+#include "ReactionStats.hpp"
+
+#include "extension/Configuration.hpp"
+
+#include "message/support/nuclear/ReactionStatistics.hpp"
+
+namespace module::support {
+
+    using extension::Configuration;
+    using NUClear::message::ReactionStatistics;
+    using ReactionStatisticsProto = message::support::nuclear::ReactionStatistics;
+
+    ReactionStats::ReactionStats(std::unique_ptr<NUClear::Environment> environment) : Reactor(std::move(environment)) {
+
+        on<Configuration>("ReactionStats.yaml").then([this](const Configuration& config) {
+            // Use configuration here from file ReactionStats.yaml
+        });
+
+        on<Trigger<ReactionStatistics>>().then([this](const ReactionStatistics& stats) {
+            auto reactionData               = std::make_unique<ReactionStatisticsProto>();
+            reactionData->name              = stats.identifier[0];
+            reactionData->trigger_name      = stats.identifier[1];
+            reactionData->function_name     = stats.identifier[2];
+            reactionData->reaction_id       = stats.reaction_id;
+            reactionData->task_id           = stats.task_id;
+            reactionData->cause_reaction_id = stats.cause_reaction_id;
+            reactionData->cause_task_id     = stats.cause_task_id;
+            reactionData->emitted           = 0;
+            reactionData->started           = 0;
+            reactionData->finished          = 0;
+            // TODO fix this
+            /*reactionData.emitted           = getUtcTimestamp<std::chrono::microseconds>(stats.emitted);
+            reactionData.started           = getUtcTimestamp<std::chrono::microseconds>(stats.started);
+            reactionData.finished          = getUtcTimestamp<std::chrono::microseconds>(stats.finished);*/
+
+            emit(reactionData);
```

Comment:
It might be better here to just directly emit to NUsight?
Not 100% sure on this one as the NetworkForwarder could also do it of course

Reply:
I thought that this was the only way to do it, what are you suggesting sorry?

Reply:
I'm flip-flopping on if it's better to use network forwarder here :P
But basically, instead of this you could do `emit<Scope::NETWORK>(reactionData, "nusight")`; and send it straight to NUsight (that's how network forwarder works internally)

Reply:
Gotcha, I think that makes sense, because the only reason you would include this module is if you were planning on using it on NUsight - it doesn't have any other purpose

Reply:
If in the future it is needed by some other module it could be changed back

Reply:
The reason I keep flip-flopping is that it lets you control all the communication to NUsight from the NetworkForwarder module. So maybe it's better to do it this way. Now that I think about it I think that's the same thing I do in the Overview module. I send it on so the network forwarder can deal with it.

Reply:
Yeah okay fair, keep it consistent